lib/chef/resource/sysctl.rb

```diff
@@ -45,6 +45,10 @@ class Sysctl < Chef::Resource
                coerce: proc { |v| coerce_value(v) },
                required: true
 
+      property :comment, [Array, String],
+               description: "Optional comments, placed above the resource setting in the generated file. For multi-line comments, use an array of strings, one per line.",
```

Comment:
the only change in the last commit is to make this description longer. Not sure why the tests are now failing?